RELEASE_NOTES.md

```diff
@@ -4,34 +4,304 @@
+
+`Enabled by default`: True
+
+`Autocorrects`: No
+
 #### Chef/PowershellScriptExpandArchive
 
+The `PowershellScriptExpandArchive` op detects the usage of the `powershell_script` resource to run the `Expand-Archive` Cmdlet. The [archive_file](https://docs.chef.io/resource_archive_file.html) resource in Chef Infra Client 15.0 should be used instead.
+
+`Examples`
+
+powershell_script using Expand-Archive to setup a website:
+
+```ruby
+powershell_script 'Expand website' do
+  code 'Expand-Archive "C:\\file.zip" -DestinationPath "C:\\inetpub\\wwwroot\\mysite"'
+  not_if { File.exist?("C:\\inetpub\\wwwroot\\mysite") }
+end
+```
+
+The same archive handled by archive_file:
+
+```ruby
+archive_file 'Expand website' do
+  path 'C:\file.zip'
+  destination 'C:\inetpub\wwwroot\mysite'
+end
+```
+
+`Enabled by default`: True
+
+`Autocorrects`: No
+
 #### Chef/PowershellInstallPackage
 
+The `PowershellInstallPackage` cop detects the usage of the `powershell_script` resource to run the `Install-Package` Cmdlet. The [powershell_package](https://docs.chef.io/resource_powershell_package.html) resources should be used to install packages via the PowerShell Package Manager instead.
+
+```ruby
+powershell_script 'Install Docker' do
+  code 'Install-Package -Name docker'
+  not_if { File.exist?("C:\\Program Files\\Docker") }
+end
+```
+
+The same package installed with powershell_package:
+
+```ruby
+powershell_package 'Install Docker' do
+  package_name 'Docker'
+end
+```
+
+`Enabled by default`: True
+
+`Autocorrects`: No
+
 #### Chef/PowershellInstallWindowsFeature
 
+The `PowershellInstallWindowsFeature` cop detects the usage of the `powershell_script` resource to run the `Install-WindowsFeature` or `Add-WindowsFeature` Cmdlets. The `windows_feature` resource should be used to install Windows features.
+
+`Examples`
+
+powershell_script using Install-WindowsFeature to install a feature
+
+```ruby
+powershell_script 'Install Feature' do
+  code 'Install-WindowsFeature -Name "Windows-Identity-Foundation"'
+  not_if '(Get-WindowsFeature | where {$_.Name -eq "Windows-Identity-Foundation"}).IsInstalled -eq true'
+end
+```
+
+The same feature install with windows_feature:
+
+```ruby
+windows_feature 'Windows-Identity-Foundation' do
+  action :install
+  install_method :windows_feature_powershell
+end
+```
+
+`Enabled by default`: True
+
+`Autocorrects`: No
+
 #### Chef/CookbookUsesNodeSave
 
+The `CookbookUsesNodeSave` cop detects the usage of `node.save` within a cookbook. node.save is often used to ensure a run_list is saved or so that other state information is immediately available for search by other nodes in your environment. The use of node.save can be incredibly problematic and should be avoided as a run failure will still result in the node data being saved to the Chef Infra Server. If search is used to put nodes into production state this may result in non-functioning nodes being used.
```

Comment:
```suggestion
The `CookbookUsesNodeSave` cop detects the usage of `node.save` within a cookbook. node.save is often used to ensure a run_list is saved, or so that other state information is immediately available for search by other nodes in your environment. The use of node.save can be incredibly problematic and should be avoided as a run failure will still result in the node data being saved to the Chef Infra Server. If search is used to put nodes into production state, this may result in non-functioning nodes being used.
```